cmd/root.go

```diff
@@ -31,33 +25,26 @@ var rootCmd = &cobra.Command{
 	Long:    "Post CDK diff log to Github Pull Request",
 	Version: Version,
 	Run: func(cmd *cobra.Command, args []string) {
-		appConfig := &config.AppConfig{
-			LogFile:       logFile,
-			TagID:         tagID,
-			RepoName:      repoName,
-			RepoOwner:     repoOwner,
-			PullRequest:   pullRequestID,
-			DeleteComment: deleteComment,
-			GithubToken:   githubToken,
-		}
+		appConfig := &config.NotifierConfig{}
```

Comment:
would make sense if we use separate command and calling this function multiple time. How ever I decided to not introduce sub commands because that would introduce breaking change. maybe later